Mathlib/Analysis/Calculus/ParametricIntegral.lean

```diff
@@ -165,6 +166,28 @@ theorem hasFDerivAt_integral_of_dominated_loc_of_lip {F' : α → H →L[𝕜] E
   apply hasFDerivAt_integral_of_dominated_loc_of_lip' δ_pos <;> assumption
 #align has_fderiv_at_integral_of_dominated_loc_of_lip hasFDerivAt_integral_of_dominated_loc_of_lip
 
+/-- Interval version of `hasFDerivAt_integral_of_dominated_loc_of_lip` -/
```

Comment:
Can you expand the doc-string for this and the other lemma to explain what the lemma is saying? There are a lot of assumptions and it is a bit difficult to immediately see what is mathematically interesting and what is not

Reply:
Sure, done.